drafts/compound-components.md

```diff
@@ -0,0 +1,252 @@
+---
+title: The Art of Component Composition - Leveraging Compound Components in React
+domain: software-engineering-corner.zuehlke.com
+tags: reactjs, components, patterns
+cover: https://cdn.hashnode.com/res/hashnode/image/upload/v1709986147750/6D9yz4Rsi.png?auto=format
```

Comment:
it looks great, but I am not a big fan of the blue ^^ maybe something darker would work better
<img width="1624" alt="image" src="https://github.com/Zuehlke/software-engineering-corner/assets/113620162/ce337ba8-4a8f-487c-bbe3-d2f453abbb1a">

Reply:
What do you think about the new version?